packages/paste-core/components/avatar/src/index.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import {Text} from '@twilio-paste/text';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {IconSize} from '@twilio-paste/style-props';
+import {getComputedTokenNames} from './utils';
+
+export interface AvatarProps {
+  size: IconSize;
+}
+export interface AvatarImageProps {
+  src: string;
+  alt: string;
+}
+
+const AvatarContext = React.createContext<AvatarProps>({size: 'sizeIcon10'});
+
+export const AvatarImg: React.FC<AvatarImageProps> = ({src, alt, ...props}) => {
+  const {size} = React.useContext(AvatarContext);
+  return <Box {...safelySpreadBoxProps(props)} as="img" alt={alt} maxWidth="100%" src={src} size={size} />;
+};
+
+const Avatar: React.FC<AvatarProps> = ({children, size = 'sizeIcon70'}) => {
+  const computedTokenNames = getComputedTokenNames(size);
+  return (
+    <AvatarContext.Provider value={{size}}>
+      <Box
+        as="div"
+        backgroundColor="colorBackgroundPrimaryLight"
```

Comment:
Should this be customizable or nah? Like have 4 variant colors for text avatars?

I haven't ran this either, but I wonder if the color would bleed under the Avatar image in some browsers too 🤔

Reply:
For now, not. We'll likely need to work on a palette of colors for things like badges and these which aren't necessarily usage specific, but like visualization colors or something. Maybe the same palette collection as charts, maybe a sibling palette. Either way, until we have that this is "fine" for the alpha I suppose